### PR TITLE
Back 19 add ens

### PR DESCRIPTION
**Pull request by isentropy**
- added ENS
- updated README. **Foreign AMB and mediator addresses changed.**
- increased DATA token mint to 1000 ETH from 1000000 Wei

### CODE REVIEW

**Comment by hpihkala** [comment]
I've updated [the doc with contract addresses in different environments](https://docs.google.com/document/d/1tygb8_9Y5AmzHzio3s3lM0rw_APGFT9hnNNutpUjO8c/edit#) to reflect the changes.

> increased DATA token mint to 1000 ETH from 1000000 Wei

Btw I didn't understand this sentence - how can DATA token mint be 1000 ETH? Did you mean you're minting 1000 DATA to some test addresses?

File: index.js [hunk 1]
```diff
@@ -119,11 +139,12 @@ async function smartContractInitialization() {
     const tokenDeployer2 = new ContractFactory(TokenJson.abi, TokenJson.bytecode, wallet)
     const tokenDeployTx2 = await tokenDeployer2.deploy("Test OTHERcoin", "\ud83e\udd84")
     const token2 = await tokenDeployTx2.deployed()
-
-    log("Minting 1000000 tokens to following addresses:")
+    //1000 ETH tokens
```
Comment (hpihkala) [inline index.js:142]:
```suggestion
    //1000 DATA tokens
```

File: index.js [hunk 2]
```diff
@@ -174,20 +197,52 @@ async function smartContractInitialization() {
...
 
-    log(`Added liquidity to uniswap exchange: ${formatEther(amt_token)} DATAcoin, ${formatEther(amt_token2)} OTHERcoin`)
     const ethwei = parseEther("1")
     let rate = await datatokenExchange.getTokenToEthInputPrice(ethwei)
     log(`1 DATAtoken buys ${formatEther(rate)} ETH`)
     rate = await othertokenExchange.getTokenToEthInputPrice(ethwei)
     log(`1 OTHERtoken buys ${formatEther(rate)} ETH`)
 
 
+/*
```
Comment (hpihkala) [inline index.js:210]: Remove commented-out code?

**Review by hpihkala — APPROVED** [review]
LGTM but a few comments could be cleaned up

**Comment by isentropy** [comment]
Eth meaning 10^18, not the ETH token

> I've updated [the doc with contract addresses in different environments](https://docs.google.com/document/d/1tygb8_9Y5AmzHzio3s3lM0rw_APGFT9hnNNutpUjO8c/edit#) to reflect the changes.
> 
> > increased DATA token mint to 1000 ETH from 1000000 Wei
> 
> Btw I didn't understand this sentence - how can DATA token mint be 1000 ETH? Did you mean you're minting 1000 DATA?

File: index.js (hunk 2 above)
Comment (isentropy) [reply index.js:210]: OK, just wanted a reference to the code from the ENS deployment script. I'll include a link to it

**Pull request merged**